### ISSUE TITLE
Confirmed on reddit that Screenscrapers api is broken at the moment on Retropie

### ISSUE THREAD

**Issue by kgonepostl** [muldjord/skyscraper#141]
**Describe the bug**
We get an error when scraping screenscrapers.fr. I'm getting: 
> Screenscraper APIv2 returned invalid.xml" when scrapping

It appears the version 2 api beta from screenscrapers.fr is broken at the moment on your end.

**To Reproduce**
This has been confirmed by other users on reddit as well. Screenscrapers is at 40% load. So it's definitely not overloaded. I logged into my account by editing the config.ini file as well. It seems to login. Same result. The guy who was helping me on reddit posted: 

> I just tested and I am getting the same results as you on titles I have already scraped previously. Their server is at 40% capacity currently so it doesnt appear to be overworked. You may want to open a new issue on **https://github.com/muldjord/skyscraper**

We're getting "Screenscraper APIv2 returned invalid.xml" when scrapping, and so is the other user. It appears the Beta V2 api has been changed and it's not accepting your requests. I'm not a coder, but that appears to be the case. It is indeed broken though. I forced clearing the cache, and vaccumed as well. No luck. This is related to the retropie experimental package. I made sure I compiled from source, and checked if there was an update. There was no update. 

Here's the reddit post if you want more information:
https://www.reddit.com/r/RetroPie/comments/bf168e/skyscraper_is_broken/

TL;DR
Went to reddit, troublshooting, turns out that it wasn't just me. The API appears to be changed/broken. 

**Special circumstances**
no

**Terminal output**

> Screenscraper APIv2 returned invalid.xml" when scrapping

**Technical information**
 - Latest version compiled from source, 3.1.4
 - Internet connection: cable
 - OS: RetroPie
 - OS Version 4.4

**Additional context**
I'm not too familiar with Linux, but I'll help anyway that I can. I'm bored anyway. Thanks for all your hard work. :-)

**Comment by muldjord** [comment]
> It appears the version 2 api beta from screenscrapers.fr is broken at the moment on your end.

No, you got this backwards. :) It's the APIv2 on ScreenScrapers end that are having issues. It seems like they forgot to renew their ssl certificate for the website and therefore Skyscraper can't verify the authenticity of the data. Try visiting their site using a browser. You should get a certificate warning which is what I am referring to. *They* need to fix this, not me. :)

**Comment by muldjord** [comment]
It seems like they just updated their certificate. It works again now. If you are still having issues, let me know. :)

**Comment by voicenoise** [comment]
screenscraper.fr site is down (only white site) and api from thegamesdb also not working. hope it comes back one of them.
